### DISCUSSION TITLE
Получение токена из кук браузера

### DISCUSSION THREAD

**Discussion by vasiaplaton** [MarshalX/yandex-music-api#515] [Идеи]
Есть идея - обменять куки Session_id на access_token, видел похожие методы в других либах.

> Forwarded message from Dmitriy
их можно обменять на access_token, позже могу скинуть
здесь есть пример обмена: https://gist.github.com/superdima05/04601c6b15d5eeb1c376535579d08a99

**Comment by NikcN22** [comment]
Есть продвижение в реализации данной идеи?
